Vostok.Metrics.System/Host/NetworkInterfaceUsageInfo.cs

```diff
@@ -0,0 +1,13 @@
+﻿using JetBrains.Annotations;
+
+namespace Vostok.Metrics.System.Host
+{
+    [PublicAPI]
+    public class NetworkInterfaceUsageInfo
+    {
+        public string InterfaceName { get; set; }
+        public long SentBytesPerSecond { get; set; }
+        public long ReceivedBytesPerSecond { get; set; }
+        public long BandwidthBitsPerSecond { get; set; }
```

Comment:
1. Why mix two measurement units (bytes/s for activity and bits/s for interface speed) instead of just using bytes for everything since this is a public API now?
2. Is there a reason not to add lazy properties that compute utilization? We could event reuse them in internal computations by summing up `NetworkInterfaceUsageInfo`s from all interfaces and accessing these properties.